### PR TITLE
done Develop

### LINKED ISSUE #1: issue1
## tasks

- [x] 기능1
- [x] 기능2

### PR DESCRIPTION
**Pull request by yonkookim**
resolve #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    기능1!!!
```
Comment (yonggooo) [inline index.html]: 느낌표 1개만 빼주세요

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    기능2!!!
```
Comment (yonggooo) [inline index.html]: 느낌표 1개만 추가해주세요

**Review by yonggooo — CHANGES_REQUESTED** [review]

**Comment by yonkookim** [comment]
넵!!!

**Review by yonggooo — APPROVED** [review]
good job

**Pull request merged**